src/lib/qboard.ts

```diff
@@ -214,6 +219,21 @@ export default class QBoard {
     await this.history.add([this.currentObject]);
   };
 
+  dragEnter = (e: DragEvent): void =>
+    this.dropArea.classList.add("file-drop-active");
```

Comment:
top-level functions should have the () and {}? i think prettier is agnostic about this and i have an opinion but idk

Reply:
In general I avoid braces for single-line functions for a few reasons. I can change it if desired

Reply:
consistency?

Reply:
like i think other single-line functions have parentheses and {} too if they're top-level

Reply:
I don't see any like that. In fact, `selectionCreated` and `objectModified` don't have braces. What do you mean by parentheses?

Reply:
huh actually nvm, yeah, they dont have braces.

by parents i meant argument parens which this already has